### ISSUE TITLE
Have a Start Page Tab but Start Page is blacnk

### ISSUE THREAD

**Issue by SoCalBryan** [sirjuddington/SLADE#1028]
**SLADE Version**: (3.1.2 or 3.1.3 or 3.1.4)
**Operating System**: (Windows)

**Issue Details**:
So I've lost the ability to see any start page items when Slade first opens up. There's a start page tab. but all that displays is whitespace.

I lost the start page in version 3.1.3 upgraded to v3.1.4 same issue. 

Did a complete uninstall and reinstalled v3.1.2 again same issue.

Anyone got an idea why this is happening
![no slade start page](https://user-images.githubusercontent.com/47740291/52943203-787e6f00-3321-11e9-9d51-3c4783521a34.jpg)

**Comment by Gaerzi** [comment]
Open the console (Ctrl-2) and scroll up to see if there are any error messages.

**Comment by SoCalBryan** [comment]
So I checked the console and there where no error messages. 

I did notice that it listed my temp directory l:\temp as i was doing some downloading to that drive and didnt want to have to wait file transfer from the %USERPROFILE%\AppData\Local\Temp folder.

I changed it back and startup page appeared as before. 

I like to have a reassigned temp folder in my user environmental variables. It's odd that Slade needs the default folder assignment. 

I changed the temp variable several times and the startpage.htm file is placed there when the program starts but won't display it. Shouldn't matter where i assign my temp folder for the startpage to show when i start the program.

Any way solved for now but the temp folder assignment in slade shouldn't be hard coded to the %USERPROFILE%\AppData\Local\Temp folder.
